workflow-service/src/main/java/com/redhat/parodos/config/properties/LdapConnectionProperties.java

```diff
@@ -0,0 +1,36 @@
+package com.redhat.parodos.config.properties;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+
+@ConfigurationProperties(prefix = "spring.ldap.connection")
+@ConfigurationPropertiesScan
+public class LdapConnectionProperties {
+
+	@Getter
+	@Setter
```

Comment:
Question: Do you use `@Setter` and `@Getter` instead of `@Data` because you want **both** getter and setter methods to be package private?
In addition, is there any need to support `toString` and `equals`?

Reply:
toString or equals are not needed, the @Data I didn't know about it!